docs/best-practices/index.md

```diff
@@ -168,87 +168,92 @@ resource "pingone_population" "my_populations" {
 
 ### Use maps with static keys when using `for_each` on resources
```

Comment:
I reordered this section because, as someone unfamiliar with the benefits mentioned, it was confusing to read the bad practice first and then identify the difference

Reply:
Makes sense!  Thanks for pointing that out